docker-compose.yml

```diff
@@ -18,16 +18,18 @@ x-environment:
   DOCKER_HOST: ${DOCKER_HOST:-missing}
 
 x-extra-hosts:
-  &default-extra-hosts
+  &default-extra-hosts # pragma: allowlist secret
   - "edx.odl.local:${OPENEDX_IP:-172.22.0.1}"
   - "host.docker.internal:host-gateway"
   - "local.edly.io:host-gateway"
 
 services:
   db:
-    image: postgres:11.6
+    image: postgres:15.7
     ports:
       - "5432"
+    environment:
```

Comment:
I was unable to run this as is running `docker compose build` then `docker compose up`. I had to add a volume:
```
    volumes:
      - db-data:/var/lib/postgresql/data
```
to the docker-compose.yml in order to get my local to not look at the previous data. I'm not sure if we want to note that or add it into the docker compose, but when I take it out, mine looks back at the old data directory.